### PR TITLE
Exclude anonymous users from accounts publication

### PR DESCRIPTION
**Pull request by priezz**
Exclude anonymous users from accounts publication when logged in as admin

### CODE REVIEW

**Comment by aaronjudd** [comment]
👍

File: server/publications/collections/accounts.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import * as Collections from "/lib/collections";
 import { Reaction } from "/server/api";
+import _ from "lodash";
```
Comment (zenweasel) [inline server/publications/collections/accounts.js]: Import order, npm packages first

Comment (priezz) [inline server/publications/collections/accounts.js]: Done

**Comment by zenweasel** [comment]
Approved (once updated)

**Comment by zenweasel** [comment]
Testing this I am seeing this error in the server console:
`Exception from sub Accounts id omBLyCHcGmm5LFP5S TypeError: _.pluck is not a function`
and this error in the browser console
`Error invoking SubsManager.subscribe("Accounts", "RcktvhiypBSDce4tp"):  Internal server error`

**Comment by aaronjudd** [comment]
@zenweasel resolved with a commit direct to development ;-)

_.pluck is deprecated in lodash 4.x and instead uses _.map.   The load order of the npm import probably brought this to light,  as I think _ (underscore) is still being exported by Meteor...?..

**Comment by priezz** [comment]
Right, I think it was underscore which worked for me, not lodash... This commit was put without the development branch was broken at the time of commit. @aaronjudd, thanks for your quick fix!

**Pull request merged**